yetanotherco/aligned_layer
# feat: add a deposit to batcher method in servicemanager

# Description

- Adds a `depositToBatcher` function for a user to fund a batcher.

# To Test

- First, run:
```
cast send <aligned_layer_service_manager_addr> "depositToBatcher(address)" <batcher_addr> --value 1ether --private-key <private_key>
```
- Then, check the balance:
```
cast call <aligned_layer_service_manager_addr> "balanceOf(address)(uint256)" <batcher_addr>
```
- Also, try sending directly without calling `depositToBatcher`:
```
cast send <aligned_layer_service_manager_addr> --value 1ether --private-key <private_key>
```
- Then, check the balance:
```
cast call <aligned_layer_service_manager_addr> "balanceOf(address)(uint256)" <batcher_addr>
```
